modules/openapi-generator/src/main/resources/python/python-experimental/model_utils.mustache

```diff
@@ -1098,6 +1112,16 @@ def get_anyof_instances(self, model_args, constant_args):
         return anyof_instances
 
     for anyof_class in self._composed_schemas['anyOf']:
+        # The composed oneOf schema allows the 'null' type and the input data
+        # is the null value. This is a OAS >= 3.1 feature.
+        if anyof_class is none_type:
```

Comment:
We will only reach this code if we are deserializing a dict payload.
This is because the `__new__` method is run before the `__init__` method. And in `__new__` we return None rather than an instance of the composed class when we pass in None and oneOf contains None type.
So in that case, get_anyof_instances is never called. So in this loop, all we need to do is skip over the none_types.
Can you change this if block to be only this?
```
if anyof_class is none_type:
  # skip none_types because we are deserializing dict data. none_type deserialization is handled in the __new__ method
  continue
```